test/e2e/performance_test.go

```diff
@@ -270,31 +338,74 @@ func networkPolicyRealize(policyRules int, data *TestData, b *testing.B) {
 	}
 }
 
+// cnpRealize runs a benchmark to measure how long it takes for <numCNPs> CNPs each with <policyRules> CIDR rules
+// to be realized as OVS flows. In order to have entities for the Network Policy to be applied to, we create two dummy
+// Pods with the "antrea-perf-test" app label, but they do not generate any traffic.
+func cnpRealize(policyRules, numCNPs int, data *TestData, b *testing.B) {
+	setupTestPods(data, b)
+	for i := 0; i < b.N; i++ {
+		cnps := generateWorkloadCNPs(policyRules, numCNPs)
+		go func() {
+			err := populateCNPs(cnps, data)
+			if err != nil {
+				b.Fatalf("Error when populating workload cnp: %v", err)
```

Comment:
maybe use "CNPs" consistently in all comments and log messages

Reply:
Done.